### ISSUE TITLE
Ubuntu 17.10 pamusb Installation?

### ISSUE THREAD

**Issue by Stevelande** [aluzzardi/pam_usb#41]
Is it possible to install pamusb under Ubuntu 17.10? Is there a working manual?

**Comment by promasu** [comment]
I'm currently working on making an PPA Package. If that is finished, I will post another comment down below.

**Comment by Stevelande** [comment]
Any News?

**Comment by promasu** [comment]
https://launchpad.net/~promasu/+archive/ubuntu/libpam-usb

**Comment by promasu** [comment]
Yes. I forgot about this issue here... You can use the ppa there.
